src/styles/blocks/catalog.scss

```diff
@@ -17,6 +17,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms ease;
```

Comment:
Consider specifying the exact properties you want to transition instead of using `all` for better performance, e.g., `transition: transform 300ms ease, color 300ms ease;`